content/glossary/dapp-developer.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Dapp Developer
+description: A developer who is building a decentralized application
+tags:
+  - fundamental
+---
+
+(This is the same as the DApp Operator definition -- let's clarify the difference between the two before adding this)A Dapp Developer takes care of deploying and operating a [Decentralized App]({{< relref "dapp" >}}).
```

Comment:
In case it's useful for this discussion, here's how I'm conceiving of the difference between the two—which I think lines up with the whitepaper:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/84613835/162027046-7e0cea42-2d2e-417f-9fee-a173d86eb887.png">

**Dapp Developers are** going to need Watcher data to power their Dapps. They'll use the Laconic SDK to query Watchers.

**Dapp Operators are** going one step deeper, and will use the Laconic SDK to write (and publish) Watchers and possibly even dig into the Laconic Stack so they can run those Watchers as Service Providers.

So my understanding is the full progression in the Laconic Ecosystem would be: 
Dapp Developer ➡️ Dapp Operator ➡️ Service Provider ➡️ Prospective Member (on the incentivized testnet) ➡️ Active Member (and thus, Validator)

Reply:
@depatchedmode This is a cool diagram! It would be great to have something like this on our site that describes the Laconic components. Perhaps something like how Kubernetes does it?
https://kubernetes.io/docs/concepts/overview/components/

Reply:
In the [branch](https://github.com/LaconicNetwork/Laconic-Documentation/pull/40) where I'm building out the site content a left a placeholder where we might do something like this, although it might be a bit early for the first iteration of the site.
https://bafybeiatwpdujx5j2pl3466fexy4qu4jhekgbmkbgiac5da6frgmkxathu.on.fleek.co/concepts/